crates/holochain/tests/tests/websocket/mod.rs

```diff
@@ -774,6 +774,7 @@ async fn concurrent_install_dna() {
 }
 
 #[tokio::test(flavor = "multi_thread")]
+#[ignore = "TODO FIXME k2 memory transport no longer supports network stats"]
 async fn network_stats() {
```

Comment:
Need to fix this, either by adding stats to the memory transport, or figuring out a good way to run sweettests with the real transport.

Reply:
... but not in this PR